lib/vfscore/automount.c

```diff
@@ -346,10 +350,86 @@ extern const char vfscore_einitrd_start[];
 extern const char vfscore_einitrd_end;
 #endif /* CONFIG_LIBVFSCORE_AUTOMOUNT_EINITRD */
 
+#define LIBVFSCORE_EXTRACTOPT_MOUNT_RAMFS		(0x1 << 0)
```

Comment:
Perhaps we could use `UK_BIT` macro here?

Reply:
We could, but... I find this more readable and direct. 🤔